search-app/docs/api.md

```diff
@@ -1387,6 +1388,10 @@ Supports ignore_case and the following aliases for "NEAR\_REAL\_TIME": "near\_re
 
      curl "%CMR-ENDPOINT%/collections?collection_data_type\[\]=NEAR_REAL_TIME&collection_data_type\[\]=OTHER"
 
+#### <a name="c-granule-data-format"></a> Find collections by format of data in granules
```

Comment:
You didn't actually implement the full support for this. See my comment on missing the `granule-data-format.lowercase` field in collection index mapping. I think you should take this out.